src/components/side-bar/side-bar.js

```diff
@@ -0,0 +1,16 @@
+import { ImgWr, SideBarWr, UserImgWr } from "./side-bar.styled";
+import Person from "../../assets/images/person.png"
+import Shape from "../../assets/images/shape.svg"
+
+export const SideBar = () => {
+  return (
+    <SideBarWr>
```

Comment:
Wr'ni to'liq Wrapper qilib yozganiz yaxshi, ertaga sizdan kn keladigan dasturchi boshini qichib o'trimaydi)